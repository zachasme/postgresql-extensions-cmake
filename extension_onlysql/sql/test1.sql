SELECT onlysql(1, 2);
