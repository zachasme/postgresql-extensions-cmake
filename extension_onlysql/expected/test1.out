SELECT onlysql(1, 2);
 onlysql 
---------
       3
(1 row)

